src/index.js

```diff
@@ -53,14 +53,19 @@ export default class extends three.Sprite {
     const font = `${this.fontWeight} ${this.fontSize}px ${this.fontFace}`;
 
     ctx.font = font;
-    const textWidth = ctx.measureText(this.text).width;
+	
+	var lines = this._text.split('\n');
```

Comment:
Please use ES6 syntax (`const`, lambda fns (`=>`)) to remain idiomatic with the existing code.